### ISSUE TITLE
WHERE clause with multiple conditions returns nothing

### ISSUE THREAD

**Issue by jeff-tenhave** [FeatureServer/winnow#23]
When using a WHERE clause that has multiple conditions, nothing will be returned. Example:

```
const feature = {
            "type": "Feature",
            "geometry": {
                "type": "Point",
                "coordinates": [-73.5, 45.5]
            },
            "properties": {
                "city": "montreal",
                "province": "quebec"
            }
 }

const options = {
        where: "city='montreal' AND province='quebec'",
};

const result = winnow.query([feature], options); // Returns nothing
```

Stepping through the query creation shows that the where clause is being created as:

 "properties->`city` = 'montreal' AND province='quebec'". 

I think the problem might be in the _pad()_ method in where.js. It doesn't seem to be able to handle more than one condition.

**Comment by dmfenton** [comment]
Thanks for reporting. We'll take a look.

**Comment by tomtom92** [comment]
@jeff-tenhave This issue should be resolved in 1.10.7.

**Comment by jeff-tenhave** [comment]
Thanks for the quick fix!
